library/src/main/java/hotchemi/android/rate/AppRate.java

```diff
@@ -54,7 +54,7 @@ public static boolean showRateDialogIfMeetsConditions(Activity activity) {
     }
 
     private static boolean isOverDate(long targetDate, int threshold) {
```

Comment:
I guess changing the type of second argument as `long` is simpler solution, what do you think?

Reply:
That's good! I will fix that way.